src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/VideoLinkAppointment.kt

```diff
@@ -23,14 +20,6 @@ enum class HearingType {
 data class VideoLinkAppointment(
   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY) val id: Long? = null,
-  val bookingId: Long,
-  val appointmentId: Long,
-  val court: String,
 
-  @Enumerated(EnumType.STRING)
-  val hearingType: HearingType,
-
-  @CreatedBy
-  var createdByUsername: String? = null,
-  val madeByTheCourt: Boolean? = true
+  val appointmentId: Long
```

Comment:
cool so leaving so can potentially add start/end times etc.. if in the future this becomes the master of VLB data